src/main/java/ch/njol/skript/conditions/CondIsStackable.java

```diff
@@ -0,0 +1,47 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+
+package ch.njol.skript.conditions;
+import ch.njol.skript.aliases.ItemType;
+import ch.njol.skript.conditions.base.PropertyCondition;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+
+
+@Name("is Stackable")
+@Description("Check if item is stackable.")
+@Examples({"diamond axe is stackable","birch wood is stackable"})
+@Since("INSERT VERSION")
+
+
+public class CondIsStackable extends PropertyCondition<ItemType> {
```

Comment:
I think using ItemStack is a better option as most syntaxes return as an ItemStack. It doesn't matter in terms of how this condition functions, as converters will make sure it's the correct type either way, but having this as ItemStack instead of ItemType will mean in most cases it won't need to convert.

Reply:
Changed already